cmake/Scripts/GenerateSleefHeader.cmake

```diff
@@ -0,0 +1,40 @@
+include(${LOCATIONS_FILE})
+
+function(run_mkrename string_params result_variable)
+  # Convert string in param variable to list to pass to the executable
+  string(REPLACE " " ";" list_params ${string_params})
+  execute_process(
+      COMMAND ${LOCATION_RUNTIME_DIR}/${TARGET_MKRENAME} ${list_params}
+      OUTPUT_VARIABLE _TMP_OUTPUT)
+  set(${result_variable} ${_TMP_OUTPUT} PARENT_SCOPE)
+endfunction()
+
+# Workaround: temporary copy of sleeflibm.h.org
+set(SLEEF_ORG_HEADER ${LOCATION_SOURCE_DIR}/src/libm/sleeflibm.h.org)
+set(SLEEF_GEN_HEADER ${LOCATION_BINARY_DIR}/include/sleef.h)
+configure_file(${SLEEF_ORG_HEADER} ${SLEEF_GEN_HEADER} COPYONLY)
+
+if(OPTION_SHOW_CONFIG)
+  message(STATUS "Generating ${SLEEF_GEN_HEADER} from: ${SLEEF_ORG_HEADER}")
+endif(OPTION_SHOW_CONFIG)
+
+# Generate header for x86
+if(SLEEF_ARCH_X86)
```

Comment:
It's also easier to have to tell the script that I want to generate the header version for x86 architecture. Currently, sleef.h can only be one of 4 versions: for x86, for aarch64, for non aarch64 arm architecture and then the default. I think I prefer to keep it like this for now, until after adding the rest of the vector extensions.